test/bdd/features/aries_router_e2e_rest.feature

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@aries_router_rest
```

Comment:
"aries_router_controller" ? just to maintain same naming convention like other tests

Reply:
Since controller can be rest or wasm, I named it as rest. But looks like our bdd tests refer controller for rest api tests - updated the name to controller for consistency.